OrbitGl/CaptureWindow.cpp

```diff
@@ -544,7 +544,7 @@ void CaptureWindow::DrawScreenSpace() {
     Box box(Vec2(0, time_graph_.GetLayout().GetSliderWidth()),
             Vec2(getWidth(), time_graph_.GetLayout().GetTimeBarHeight()),
             GlCanvas::kZValueTimeBarBg);
-    ui_batcher_.AddBox(box, Color(70, 70, 70, 200));
+    ui_batcher_.AddBox(box, Color(67, 67, 67, 200));
```

Comment:
Should this be a defined constant of the class to make it clear that this and the color above have to match (and will match in the future)?

Reply:
Good point, I introduced GlCanvas::kBackgroundColor (not in CaptureWindow, because all other constants are defined in GlCanvas - Track uses the same color and already includes constants from GlCanvas, so this seemed to be the better place). The timebar is using the same color with a different alpha value.